pyproject.toml

```diff
@@ -24,6 +24,7 @@ extension-pkg-whitelist = [
     "pydantic"
 ]
 load-plugins = "pylint_fixme_info"
+max-branches = 20
```

Comment:
I think it may be better to just disable it for the function(s) in question.  Generally I'd suspect functions requiring this should be redesigned a bit, so better to flag it and require the developer to ack it if they're gonna ignore it :D